app/models/category.rb

```diff
@@ -265,6 +265,15 @@ def description_text
     end
   end
 
+  def access_category_via_group
+    Group
+      .joins(:category_groups)
+      .where("category_groups.category_id = ?", self.id)
+      .where("groups.public_admission OR groups.allow_membership_requests")
+      .order(:allow_membership_requests)
```

Comment:
Sorting by a boolean is a little odd.

Reply:
It is done this way to favor groups with public admission over groups with membership requests.